packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -246,11 +254,12 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
         assert.strictEqual(bufferToString(await (dataStore._root.get("my blob")).get(), "utf-8"), text);
 
         const attachP = container.attach(provider.driver.createCreateNewRequest(provider.documentId));
-        if (provider.driver.type !== "odsp") {
```

Comment:
i don't think this should be modified. just leave it as is, and let whoever fixes it deal with it later